zerver/lib/notifications.py

```diff
@@ -113,6 +113,13 @@ def relative_to_full_url(content):
             user_profile.realm.uri + r"/static/generated/emoji/images/emoji/",
             content)
 
+        # Stream links need to be converted from relative to absolute. They
+        # have href values in the form of "/#narrow/stream/...".
+        content = re.sub(
+            r"/#narrow/stream/",
+            user_profile.realm.uri + r"/#narrow/stream/",
+            content)
+
```

Comment:
hmm, maybe worth seeing if we should be using that function in more places is the right fix here?